lib/network/Images.js

```diff
@@ -52,6 +60,13 @@ Images.prototype.load = function(url, brokenUrl) {
           me.callback(this);
         }
       }
+      else if (this.src === me._resolveRelativeUrl(brokenUrl)){
```

Comment:
I dont really agree with this method.. But the problem is relevant. I'll push the fix below to develop:

```
else if (me.imageBroken[url] === true) {
        console.error("Could not load brokenImage:", brokenUrl);
        delete this.src;
        if (me.callback) {
          me.callback(this);
        }
      }
      else {
        this.src = brokenUrl;
        me.imageBroken[url] = true;
      }
```